Sources/SPMBuildCore/PluginInvocation.swift

```diff
@@ -493,16 +499,14 @@ public struct PluginInvocationTestResult: Encodable {
             public var tests: [Test]
             public struct Test: Encodable {
                 public var name: String
-                public var output: String
-                public var status: Status
+                public var outcome: Outcome
                 public var duration: Double
-                public enum Status: String, CaseIterable, Encodable {
+                public enum Outcome: String, Encodable {
```

Comment:
Result?

Reply:
I had that initially, actually, but avoid it in the proposal since result is also the name of the outer structure.  But "result" seems like a better name even so.  This matches the proposal so if we change it that will need to be an amendment.